.github/workflows/continuous-integration.yml

```diff
@@ -80,9 +69,6 @@ jobs:
       - name: Check Types 📚
         run: make types
 
-      - name: Test Docs 🕸
-        run: make test-docs
-
```

Comment:
can you remove [these 2 make targets](https://github.com/RasaHQ/rasa-sdk/blob/main/Makefile#L58-L67) too? I think they're unused now